R/accessory.R

```diff
@@ -913,7 +913,7 @@ read_contrasts <-
     # Read the contrasts
 
     contrasts <- read_metadata(filename)
-    contrast_cols <- c(variable_column, reference_column, target_column, blocking_column)
+    contrast_cols <- c(variable_column, reference_column, target_column)
```

Comment:
You're also going to want to make the stuff below that uses the blocking stuff conditional. I've noticed that I've used the literal 'blocking', which was bad, it should refer to `blocking_column`.

I suggest that where `blocking_col` is not in the columns of the input table, you just set the column to an empty string:

```
contrasts[[blocking_col]] <- ''
```

Reply:
yeah I also thought the additional blocking code should be conditional, but weirdly enough, the code worked without making that part conditional (even when no blocking was in the input). But I added the conditional